assets/src/styles/components/ImageHoverButtons.scss

```diff
@@ -0,0 +1,48 @@
+.buttons-overlay {
+  position: absolute;
+  z-index: 2;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  display: flex;
+
+  button {
+    display: none;
+  }
+
+  &:hover {
+    background: rgba(255, 255, 255, 0.3);
+
+    button {
+      display: inline-flex;
+    }
+  }
+
+  .block-style-icons & {
+    justify-content: flex-start;
+    margin-left: 10px;
```

Comment:
We could ask to the design team if they prefer to use `$sp-1x` instead and start standardizing our spacing values to p4 blocks.

What do you think?

Reply:
That's a good point! I've replaced those by `$sp-1` and also used `margin-inline-start` instead of `margin-left` for RTL sites 👌